app/services/offender_service.rb

```diff
@@ -73,13 +73,9 @@ def get_offenders_for_prison(page_number:, page_size:)
       mapped_tiers = CaseInformationService.get_case_information(nomis_ids)
 
       offenders.select { |offender|
```

Comment:
This can now be an 'each' - a bit naughty of it to be a select that changes the iterable...